src/clipper/clipper.cpp

```diff
@@ -2013,7 +2013,7 @@ void Clipper::InsertLocalMinimaIntoAEL(const cInt botY)
 
     //if any output polygons share an edge, they'll need joining later ...
     if (Op1 && IsHorizontal(*rb) && 
-      m_GhostJoins.size() > 0 && (rb->WindDelta != 0))
+      !m_GhostJoins.empty() && (rb->WindDelta != 0))
```

Comment:
This file comes from the [polyclipping](https://sourceforge.net/p/polyclipping/) library. In the past, we’ve [attempted](https://github.com/mapbox/mapbox-gl-native/pull/1425#commitcomment-11049117) to keep this file in sync with the upstream project.